src/telliot_feed_examples/cli.py

```diff
@@ -62,9 +62,23 @@
     type=int,
     default=500000,
 )
+@click.option(
+    "--rpc-url",
+    "-rpc",
+    "override_rpc_url",
+    help="override the config RPC url. requires also overriding the chain ID",
```

Comment:
I'd remove the `. requires also overriding...` part, as it does this automatically behind the scenes it looks like.

Reply:
I agree